packages/react-error-overlay/src/overlay.js

```diff
@@ -156,6 +160,10 @@ function crash(error: Error, unhandledRejection = false) {
   }
   consumeError(error, unhandledRejection, CONTEXT_SIZE)
     .then(ref => {
+      if (ref == null) {
+        console.warn('react-error-overlay ignored:', error);
```

Comment:
We should be careful not to leave messages that would be confusing to users. If one sees this message, is this a bug? Should they file an issue? Should the message say so?